docs/development_suite/set-up-infrastructure/enhanced-project-workflow.md

```diff
@@ -87,6 +81,16 @@ Versions can only be deleted by Project Administrators.
 
 ![Versions management page](img/versions-overview.png)
 
+### Revision history
+
+To keep track of the Project evolution, the user can create `versions` (similar to Git tags) that act as immutable snapshots of the configuration in a particular moment. To keep track of how a revision has changed over time, the user can access its revision history, which contains past snapshots of the Project configuration.
```

Comment:
We start the paragraph for revision history speaking about versions; that sounds odd.

I'd keep this paragraph in its previous location or rephrase it to emphasize revision history

```suggestion
To keep track of how a revision has changed over time, the user can access its revision history, which contains past snapshots of the Project configuration.
```

And put somewhere else the sentence 

```
To keep track of the Project evolution, the user can create `versions` (similar to Git tags) that act as immutable snapshots of the configuration in a particular moment. 
```

Reply:
Description of `version` has been moved to the Version paragraph describing the Versions Management page. Let me know if it looks better.

Reply:
sounds great. I've removed references to git tags.